projects/sonar/src/app/record/document/detail/detail.component.ts

```diff
@@ -14,27 +14,156 @@
  * You should have received a copy of the GNU Affero General Public License
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { Observable } from 'rxjs';
 
 @Component({
-  templateUrl: './detail.component.html'
+  templateUrl: './detail.component.html',
 })
-export class DetailComponent {
+export class DetailComponent implements OnInit {
   /** Observable resolving record data */
   record$: Observable<any>;
 
   /** File key to preview */
   previewFileKey: string;
 
+  // Show only three creators on startup.
+  creatorsLength = 3;
+
+  // Record retrieved from observable.
+  record: any = null;
+
+  // List of creators
+  creators: Array<any> = [];
+
+  /**
+   * Component initialisation.
+   *
+   * Retrieve record from observable.
+   */
+  ngOnInit() {
+    this.record$.subscribe((record: any) => {
```

Comment:
Maybe you need to implements ngOnDestroy and unsubscribe this?

Reply:
No need to unsubscribe as this observable emit only one value and complete after.